lib/telegram_mailer_patch.rb

```diff
@@ -106,8 +105,13 @@ def issue_add_with_telegram(issue, to_users, cc_users)
         :short => true
       } if Setting.plugin_redmine_telegram[:display_watchers] == 'yes'
 
-
-      Mailer.speak(msg, channel, attachment, token)      
+      users.each do |user|
```

Comment:
Use `users` - not good idea. I have >100 users in Redmine. And if they will receive messages on each change of tickets - it will be SPAM.
In this context we have `to_users` and `cc_users` - try to send messages for they.